environments/sre/prometheusrules.jsonnet

```diff
@@ -161,4 +162,30 @@ local capitalize(str) = std.asciiUpper(std.substr(str, 0, 1)) + std.asciiLower(s
 
     spec: alerts.prometheusAlerts,
   },
+  'observatorium-jaeger-production.prometheusrules': {
+    apiVersion: 'monitoring.coreos.com/v1',
+    kind: 'PrometheusRule',
+    metadata: {
+      name: 'observatorium-jaeger-production',
```

Comment:
I don't think we need this `-production` separation anymore, as the clusters are separate now. We can simply call it observatorium-jaeger :)